packages/web-config-server/src/apiV1/organisationUnit.js

```diff
@@ -50,11 +55,8 @@ export default class extends RouteHandler {
           },
         })
       : [];
-    const orgUnitHierarchy = await this.filterForAccess([
-      projectEntity,
-      country,
-      ...countryDescendants,
-    ]);
```

Comment:
Removed this permissions check here, as for all entities other than the country entity we were immediately returning `true`. Instead we just check the country entity at the start and return an empty object if we don't have access to that country.